lib/Ed25519KeyPair.js

```diff
@@ -183,17 +255,19 @@ class Ed25519KeyPair extends LDKeyPair {
   }
 
   /**
-   * Generates and returns a Multiformat encoded ed25519 public key fingerprint
-   * (for use with cryptonyms, for example).
-   *
+   * @description Generates and returns a multi-format encoded
+   * ed25519 public key fingerprint (for use with cryptonyms, for example).
+   * @example
+   * edKeyPair.fingerprint();
+   * // dfdsfdsfds3432423
    * @see https://github.com/multiformats/multicodec
    *
-   * @returns {string}
+   * @returns {string} A verifiable cryptographic signature.
    */
   fingerprint() {
     const buffer = new forge.util.createBuffer();
 
-    // ed25519 cryptonyms are multicodec encoded values, specifically:
+    // ed25519 cryptonyms are multi-codec encoded values, specifically:
```

Comment:
These [multiformats](https://github.com/multiformats/multiformats) references shouldn't have hyphens.

Reply:
- [ ] change multi-formats to multiformats spell checker be damned.

Reply:
I believe this is resolved, but there are other hyphenated words in the doc that might actually be npm packages. I also chanced multi-codec to multicodec as multicodec is the name of the npm package used to generate some keys. I quick grep -nr '-' ./lib
reveals that:
RSASSA-PSS
is still hyphenated so mark as resolved if that is ok no multi-codec or multi-formats out there anymore.